apps/client/package.json

```diff
@@ -25,14 +25,13 @@
     "lodash.isboolean": "^3.0.3",
     "lodash.isnumber": "^3.0.3",
     "lodash.isstring": "^4.0.1",
-    "lodash.uniqby": "^4.7.0",
     "named-urls": "^2.0.0",
     "react": "^16.13.1",
     "react-app-polyfill": "^1.0.6",
     "react-device-detect": "^1.12.1",
     "react-dom": "^16.13.1",
     "react-helmet": "^6.1.0",
-    "react-hook-form": "^5.7.2",
+    "react-hook-form": "^6.0.0",
```

Comment:
Why is this in the uniqBy pr?

Reply:
Because this is a pr to make the bundle sizes smaller. This makes the bundles size a lot smaller.

Reply:
Cool!